unified-codes/tools/scripts/keycloak/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo " * Launching keycloak container..."
+docker run -p 9990:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin jboss/keycloak
```

Comment:
splitting this out makes it problematic to use - the docker instance doesn't retain data, so the `setup.sh` script always needs to be run.
When split like this - you need to run `run.sh` and `setup.sh`
I see what you're doing, and agree that running and setting up should be different functions. Until we resolve the retention of data when restarting docker instances of keycloak, implementing the separation is just creating a point of failure.

Reply:
My thinking is that in actual production we will probably have to update these scripts anyway, so it might be good to have a consistent script standard? Just IMO, having them separate still allows for `run.sh && setup.sh`, as well as running them separately? Would the possible point of failure (I assume someone accidentally wiping any additional test data?) be a worthwhile trade off for the flexibility/consistency (at least for now?).

Reply:
yeah, I think the trade off is worth it. has to be split some time, may as well be now.